### DISCUSSION TITLE
Syntax error while run ww3_shel

### DISCUSSION THREAD

**Discussion by DiegoGutierrez2210** [NOAA-EMC/WW3#728] [Q&A] [answered]
Hi guys

I need some help to solve an issue about this:
**forrtl: severe (59): list-directed I/O syntax error, unit 10, file /home/ww3_516/forecast/pacific/./ww3_shel.inp**
When i excecute:
/home/ww3_516/WW3/WW3-6.07.1/model/exe_pacific/ww3_shel > ww3_shel.out
Any idea about this error? I know that my data input is correct, and I think the problem is about my shel script. I share my INP and OUT script.
[ww3_shel.out.txt](https://github.com/NOAA-EMC/WW3/files/8934907/ww3_shel.out.txt)
[ww3_shel.inp.txt](https://github.com/NOAA-EMC/WW3/files/8934909/ww3_shel.inp.txt)
Switch: F90 SHRD NOGRB OMPG OMPX NC4 PR3 UQ FLX2 LN1 ST6 NL1 BT1 DB1 MLIM TR1 BS0 IC0 IS0 REF1 XX0 WNT1 WNX1 CRT1 CRX1 TRKNC O0 O1 O2 O2a O2b O2c O3 O4 O5 O6 O7 O10 O11 O14
I used Intel compilers: ifort and icc 2021.6.0 20220226 for WW3v6.07.1
Regards

**Comment by ffayalac** [comment, marked as answer]
Hi, Diego. 

I strongly recommend you deactivate the options that you will not use during your simulation. Your problem occurs in the ww3_shel file because it is configured to estimate the separated wave field data (see penultimate line) but you probably do not have the correct input files to calculate them. 

`19680606 000000   0  20010102 000000`
   
Instead of 

`19680606 000000   3600  20010102 000000`
`      0 999 1 0 999 1 T`

Please see a ww3_shel file modified and its corresponding log (I killed the simulation so my interest is showing that the new file is OK)

[ww3_shel.inp.txt](https://github.com/NOAA-EMC/WW3/files/8970448/ww3_shel.inp.txt)
[shel.log](https://github.com/NOAA-EMC/WW3/files/8970447/shel.log)

**Comment by DiegoGutierrez2210** [comment]
Thanks Frankly, It's running now! thanks for the advice

Regards
